packages/core/admin/admin/src/pages/MarketplacePage/components/PluginCard/index.js

```diff
@@ -49,9 +62,16 @@ const PluginCard = ({ plugin }) => {
           height={11}
         />
         <Box paddingTop={5}>
-          <Typography as="h3" variant="delta">
-            {attributes.name}
-          </Typography>
+          <Flex alignItems="center">
+            <Typography as="h3" variant="delta">
+              {attributes.name}
+            </Typography>
+            {attributes.validated && (
+              <Tooltip description="Plugin verified by Strapi">
+                <Icon as={CheckCircle} marginLeft={2} color="success600" />
+              </Tooltip>
+            )}
```

Comment:
Should/ could this be part if the "h3"? For screenread it would read then "Title [stop] Plugin verified by Strapi" ...

Reply:
oh cool, thanks I'll update